#!/usr/bin/env python
PACKAGE = "pure_spinning_local_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("k_betta",   double_t,   1, "goal angle error gain", 0.1 )
gen.add("yaw_goal_tolerance",   double_t,   1, "radians", 0.05 )
gen.add("enable_obstacle_checking",   bool_t,   1, "bool", True )
gen.add("max_angular_z_speed",   double_t,   1, "maximum angular speed in radians per second", 1.0 )

exit(gen.generate(PACKAGE, "pure_spinning_local_planner", "PureSpinningLocalPlanner"))
